### PR TITLE
Completed LoginCard

### LINKED ISSUE #9: Login Card component
## Issue Type

- [ ] Bug (https://github.com/react-boilerplate/react-boilerplate/blob/master/.github/CONTRIBUTING.md#bug-reports)
- [x] Feature (https://github.com/react-boilerplate/react-boilerplate/blob/master/.github/CONTRIBUTING.md#feature-requests)

## Description
Implement the login card component. Use the card component from react-bootstrap. This could also implement the registration  component :) 
img_1546 2

### PR DESCRIPTION
**Pull request by Jubstron**
This pull request includes the LoginCard component which contains a form that toggles between login and register.

#codeFastAndEatAss :metal:

- [x] You have followed our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/.github/CONTRIBUTING.md)
- [x] double-check your branch is based on `dev` and targets `dev` 
- [ ] Pull request has tests (we are going for 100% coverage!)
- [ ] Code is well-commented, linted and follows project conventions
- [x] Documentation is updated (if necessary)
- [x] Internal code generators and templates are updated (if necessary)
- [x] Description explains the issue/use-case resolved and auto-closes related issues

This closes #9

### CODE REVIEW

**Review by wrejcodes — CHANGES_REQUESTED** [review]
We should be doing a confirm password input on registration and we should potentially look into doing validations for the form input. Potentially we just ignore that for now though. @Quinn-Donnelly  What do you think?

**Review by Quinn-Donnelly — CHANGES_REQUESTED** [review]
I agree with @wrejcodes

File: app/components/LoginCard/index.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  changeForm() {
+    if (this.state.toggle === 'Register') {
+      this.setState({ toggle: 'Login', current: 'Register' });
+    } else {
+      this.setState({ toggle: 'Register', current: 'Login' });
+    }
+  }
+  handleChange() {
+    const password = document.getElementById('password');
+    const confirmPassword = document.getElementById('confirmPassword');
```
Comment (Quinn-Donnelly) [inline app/components/LoginCard/index.js]: Change these to refs so we aren't searching the DOM each time they click a key.

**Review by Quinn-Donnelly — CHANGES_REQUESTED** [review]

File: app/components/LoginCard/index.js (hunk 1 above)
Comment (Jubstron) [reply app/components/LoginCard/index.js]: :anger: u r stinky :anger:

**Review by Quinn-Donnelly — APPROVED** [review]
Looks good to me.

**Review by wrejcodes — APPROVED** [review]
Looks good

**Review by Quinn-Donnelly — APPROVED** [review]
Good Catch @wrejcodes

File: app/components/LoginCard/index.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
+/**
+*
+* LoginCard
+*
```
Comment (bsonge) [inline app/components/LoginCard/index.js:4]: you should add a description block up here

**Review by bsonge — APPROVED** [review]
these changes are fresher than a cucumber

**Comment by bsonge** [comment]
this fixes #15

**Pull request merged**